airbyte-integrations/connectors/source-facebook-marketing/source_facebook_marketing/source.py

```diff
@@ -183,6 +184,12 @@ def get_custom_insights_streams(self, api: API, config: ConnectorConfig) -> List
         """return custom insights streams"""
         streams = []
         for insight in config.custom_insights or []:
+            insight_fields = set(insight.fields)
+            if insight_fields.intersection(UNSUPPORTED_FIELDS):
+                # https://github.com/airbytehq/oncall/issues/1137
+                logger.info(f"Following fields were removed from the Custom{insight.name} fields list due to error"
+                            f" on Facebook side: {insight_fields.intersection(UNSUPPORTED_FIELDS)}")
+                insight_fields -= UNSUPPORTED_FIELDS
```

Comment:
I am not sure how it works, I see you just removed this fields from some temp variable 
`insight_fields` and that's all ?

is it ok?